Task 8/UsersAndAwards(Framework)/EPAM.AwardsAndUsers.PL.WebPL/Pages/LogIn.cshtml

```diff
@@ -0,0 +1,62 @@
+﻿@using EPAM.AwardsAndUsers.PL.WebPL.Models;
+@using EPAM.AwardsAndUsers.Common.Dependencies;
+@using EPAM.AwardsAndUsers.Common.Entities;
+
+@{
+    Layout = "~/Layouts/_BasicLayout.cshtml";
+    Page.Title = "Log In";
+    FormsAuthentication.SignOut();
+    DataModel dataModel = new DataModel();
+    dataModel.WrongPassword = false;
+    InputHelper help = new InputHelper();
+
+    if (IsPost)
+    {
+        var username = Request["username"];
+        var password = Request["password"];
+
+        if (help.CheckAuthParametres(username) && help.CheckAuthParametres(password))
+        {
+            if (DependencyResolver.Instance.UsersAndAwardsBLL.FindUser(username))
+            {
+                if (DependencyResolver.Instance.UsersAndAwardsBLL.AuthUser(password.GetHashCode()))
+                {
+                    FormsAuthentication.SetAuthCookie(username, createPersistentCookie: true);
+                    Response.Redirect("~");
+                }
+                else
+                {
+                    dataModel.WrongPassword = true;
+                }
+            }
+            else
+            {
+                Response.Redirect("~/Pages/RegisterNewUser");
+            }
```

Comment:
This is not a clear UX - I don't have a user and to register I have to somehow guess that I need to try to login with auth information that doesn't actually exist.
Make a `Register` button